pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -232,6 +233,43 @@ func TestGetTransactionConfirmations_Negative_Integration(t *testing.T) {
 	})
 }
 
+func TestGetLatestBlockHeightConcurrently_Integration(t *testing.T) {
```

Comment:
Checking my reasoning: This test asserts that we no longer see the panic described in https://github.com/keep-network/keep-core/issues/3699 when multiple goroutines call the function but we expect occasional failures as described in https://github.com/checksum0/go-electrum/issues/10. Is that correct?

Reply:
Correct, this fixes https://github.com/keep-network/keep-core/issues/3699, but at the same time this test made https://github.com/checksum0/go-electrum/issues/10 issue happen way more frequently because it uses multiple goroutines.

Reply:
That sounds good to me.

Reply:
Here [9882119](https://github.com/keep-network/keep-core/pull/3701/commits/98821197ac36afa1b5b3f538a6a7a9162c22f437)